sktime/contrib/distance_based/elastic_ensemble_from_file.py

```diff
@@ -1,13 +1,21 @@
+# -*- coding: utf-8 -*-
 import numpy as np
 import os
-from sklearn.metrics.scorer import accuracy_score
```

Comment:
This wasn't working with update to scikit-learn being pinned to 0.24.*. Rest of changes to this file are from the CI auto-reformatting.

Reply:
see https://github.com/alan-turing-institute/sktime/pull/810